library.json

```diff
@@ -0,0 +1,13 @@
+{
+  "extends": [
+    "github>4Catalyzer/renovate-config",
+    "schedule:weekly"
+  ],
+  "rangeStrategy": "update-lockfile",
```

Comment:
isn't `update-lockfile` a little too nasty? doesn't it mean that eg people will not be able to use rb unless they are on latest react?

Reply:
huh? i _think_ what this means is "only update `yarn.lock`, don't update `package.json`".

which i think is what we want for prod deps – we shouldn't bump the required version in `package.json`, but we might as well run against the latest version for tests/dev/&c.

Reply:
> Update the lock file when in-range updates are available, otherwise replace for updates out of range.

ok right, makes sense, basically it honors whatever you have in package.json. unclear what would happen if you have eg. react > 14 < 17. will it widen?

Reply:
not sure!